torchci/lib/drciUtils.ts

```diff
@@ -10,9 +10,9 @@ export const OWNER: string = "pytorch";
 export const DRCI_COMMENT_START = "<!-- drci-comment-start -->\n";
 export const OH_URL =
     "https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours";
-export const DOCS_URL = "https://docs-preview.pytorch.org/";
-export const PYTHON_DOCS_URL = "/index.html";
-export const CPP_DOCS_URL = "/cppdocs/index.html";
+export const DOCS_URL = "https://docs-preview.pytorch.org";
+export const PYTHON_DOCS_PATH = "index.html";
+export const CPP_DOCS_PATH = "cppdocs/index.html";
```

Comment:
I've removed the leading and trailing `/` here, because I found it rather hard to read below when I needed a `/`.